healvis/data/configs/obsparam_test.yaml

```diff
@@ -13,9 +13,10 @@ telescope:
   telescope_location: (-30.72152777777791, 21.428305555555557, 1073.0000000093132)  # LatLonAlt location
   telescope_name: HERA
 freq:
-  Nfreqs: 3
-  start_freq: 100000000.0  # Hz
-  bandwidth:  100000000.0  # Hz
+  bandwidth: 50000000.0    # Total frequency coverage (Nfreqs * channel_width)
+  start_freq: 100000000.0   # Center of first frequency channel
+  channel_width: 5000000.0  # Separation of frequency channel centers
+  Nfreqs: 10                # Number of frequency channels
```

Comment:
I think we should specify three of the following, not four, because they can produce conflicting information if not properly set.
I think the most natural thing is `start_freq`, `bandwidth` and `Nfreqs`, as we had before. so prob we should get rid of `channel_width`. alternatively, if you prefer `channel_width`, then get rid of `bandwidth`.